packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -145,6 +144,85 @@ describe('StripeHelper', () => {
     sandbox.restore();
   });
 
+  describe('getCachedResult', () => {
+    const expectedResult = { status: 'success' };
+
+    const cacheKey = 'testKey';
+    let refreshFunction;
+
+    beforeEach(() => {
+      refreshFunction = sandbox.stub().returns(expectedResult);
+    });
+
+    it('follows read-through caching logic', async () => {
+      // On first call: cache miss, call refresh function, cache the result
+      let actualResult = await stripeHelper.getCachedResult(
+        cacheKey,
+        refreshFunction
+      );
+      assert(mockRedis.get.calledOnceWith, { cacheKey });
+      assert(mockRedis.set.calledOnceWith, { cacheKey });
```

Comment:
Doesn't the `set` call take more than one argument?  At the very least, the key and the value?

Reply:
yes, it does take the key and the value. I can add the value to the argument check

Reply:
ooh hmm, I think the reason that it was passing is because `assert(mockRedis.set.calledOnceWith, { cacheKey });` isn't actually calling `mockRedis.set.calledOnceWith`.